tests/fixtures/MakeControllerAttributes/tests/GeneratedControllerTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Tests;
+
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @author Jesse Rushlow <jr@rushlow.dev>
+ */
+class GeneratedControllerTest extends WebTestCase
+{
+    /**
+     * This tests passes as is, but I can do better.....
+     */
```

Comment:
Probably you could remove a few things from this test (like testing the path) and instead actual make a request to `/attribute` and assert that you get back a valid response.

Reply:
removed test as it is redundant